policyengine_us_data/datasets/puf/puf.py

```diff
@@ -203,15 +203,113 @@ def preprocess_puf(puf: pd.DataFrame) -> pd.DataFrame:
     # Ignore f2441 (AMT form attached)
     # Ignore cmbtp (estimate of AMT income not in AGI)
     # Ignore k1bx14s and k1bx14p (partner self-employment income included in partnership and S-corp income)
-    qbi = np.maximum(0, puf.E00900 + puf.E26270 + puf.E02100 + puf.E27200)
-    # 10.1% passthrough rate for W2 wages hits the JCT tax expenditure target for QBID
-    # https://gist.github.com/nikhilwoodruff/262c80b8b17935d6fb8544647143b854
-    W2_WAGES_SCALE = 0.101
-    puf["w2_wages_from_qualified_business"] = qbi * W2_WAGES_SCALE
-
-    # Remove aggregate records
-    puf = puf[puf.MARS != 0]
+    # --- Qualified Business Income Deduction related variables ---
+    # Income sources
+    puf["sole_proprietorship_net_income"] = puf.E00900  # Schedule C
+    puf["schedule_F_farm_net_income"] = puf.E02100  # Schedule F active farming operations 
```

Comment:
we keep all variables lowercase
```suggestion
    puf["schedule_f_farm_net_income"] = puf.E02100  # Schedule F active farming operations 
```